### DISCUSSION TITLE
Scalability of UPPAAL

### DISCUSSION THREAD

**Discussion by MathieuMarchand** [UPPAALModelChecker/UPPAAL-Meta#223] [Q&A] [answered]
Hi everyone,

I was wondering if there have been experiments or studies to have an idea on the scalability of UPPAAL.
By that, I mean for instance, if we have 100 states, it takes 100ms to verify a property, but with a 200 states system, it takes 1000ms, to get an idea of how big of a system we can build and what could be reasonable computation time to expect wrt the size of the FSM.

Thank you in advance for your answers

**Comment by mikucionisaau** [comment, marked as answer]
This paper could be starting point:
https://link.springer.com/chapter/10.1007/978-3-540-45069-6_40
It discusses various optimization techniques, so any performance study should include it in references.
